MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly

```diff
@@ -0,0 +1,105 @@
+\version "2.18.2"
+
+\header {
+  title = "Nativity of the Theotokos"
+  subtitle = "Antiphon"
+  subsubtitle = "Verse for 3rd (Sung immediately before Apolytikion)"
+  copyright = "This arrangement is free to distribute for Liturgical use"
+  tagline = "Arranged using https://oca.org/liturgics/learning-the-tones"
+}
+
+#(set-default-paper-size "letter")
+
+% Provide an easy way to group a bunch of text together on a breve
+% http://lilypond.org/doc/v2.18/Documentation/notation/working-with-ancient-music_002d_002dscenarios-and-solutions
+recite = \once \override LyricText.self-alignment-X = #-1
+
+\defineBarLine "invisible" #'("" "" "")
+global = {
+  \time 1/1 % Not used, Time_signature_engraver is removed from layout
+  \key g \major
+  \set Timing.defaultBarType = "invisible" %% Only put bar lines where I say
+}
+
+verseOne = \lyricmode {
+  Here will I dwell, for I have cho -- sen her.
+  The Most High hath hall -- ow -- ed His tab -- er -- na -- cle.
```

Comment:
hal -- lowed, I know we often split this word into 3 syllables liturgically, but here the duple meter makes "-ed" sound accented, so 2 syllables would sound slightly better. Also it just feels better to me to have it as 2 syllables when used as a verb.
tab -- er -- nac -- le.

Reply:
What lexical phoneme dictionary are you using?  we should use the same one.  I checked the dictionary splitting with: https://www.merriam-webster.com/dictionary/tabernacle

Reply:
Also relevant regarding latex: https://tex.stackexchange.com/questions/133859/using-tex-to-split-words-into-syllables